loadtest/control/simulcontroller/controller.go

```diff
@@ -170,6 +170,14 @@ func (c *SimulController) Run() {
 			run:       deletePost,
 			frequency: 0.06,
 		},
+		{
+			run:       c.updateCustomStatus,
+			frequency: 0.05,
```

Comment:
Checked Community Grafana. It has nearly the same frequency as deletePost, may be slightly slower. Hence made it to 0.05.